### DISCUSSION TITLE
IDs with Divs

### DISCUSSION THREAD

**Discussion by victorgarrido1** [twbs/bootstrap#39912] [Q&A]
Hello,

I am a relativity new developer diving toes to using toolkits like bootstrap. And this is the first time I am using the discussion board.
Is it possible to target a div attribute as with IDs alongside with bootstrap?

For example: 
`<h2 class="font-semibold text-center text-2xl lg:text-3xl">Weather Headline will go here ~~~`
Would the ID attribute work as intended with bootstrap. Should anyone looking in my question and needs more clarification, I am happy to chat as the goal is to learn the lingo.

**Comment by GeorgiosDrivas** [comment]
Hello! I would like to help you with your question.

As of my knowledge, Bootstrap does not affect ID's so if an element has both classes and id, it is perfectly fine. Keep in mind though, that Bootstrap classes will overrun any css code you will put, unless you use the "!important" property on your css rule.

I am happy to help, feel free to reply if you have further questions about this topic!
